packages/@react-aria/disclosure/test/useDisclosure.test.ts

```diff
@@ -172,8 +172,9 @@ describe('useDisclosure', () => {
     });
   
     expect(result.current.state.isExpanded).toBe(false);
-    expect(ref.current.getAttribute('hidden')).toBeNull();
-    expect(onExpandedChange).not.toHaveBeenCalled();
+    expect(ref.current.getAttribute('hidden')).toBe('until-found');
```

Comment:
I listed a lot of test cases in my review, can we add them either here or in followup?